src/components/table.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './style.css';
+
+import { Text } from './text';
+
+export type HeaderType = string[];
+export type RowType = Record<string, string>;
+
+interface TableProps {
+  header: HeaderType;
```

Comment:
```suggestion
export type RowType = Record<string, string>;

interface TableProps {
  header: string[];
```

Arrays de tipos primitivos é algo bem comum, não precisa criar um type pra eles.